sourcetracker/_cli/gibbs.py

```diff
@@ -19,11 +19,12 @@
 from ipyparallel import Client
 
 from sourcetracker._cli import cli
-from sourcetracker._sourcetracker import (biom_to_df, _gibbs, _gibbs_loo,
-                                          intersect_and_sort_samples,
+from sourcetracker._sourcetracker import (gibbs, intersect_and_sort_samples,
                                           get_samples, collapse_source_data,
-                                          subsample_dataframe)
-from sourcetracker._util import parse_sample_metadata
+                                          subsample_dataframe,
+                                          validate_gibbs_input, plot_mpm)
+
+from sourcetracker._util import parse_sample_metadata, biom_to_df
 
 
 @cli.command(name='gibbs')
```

Comment:
Can you drop the `loo` parameter from the CLI, and just use the same strategy that you use in the API (ie., use loo if only one dataframe is provided? That would be cleaner to document/describe to users.